rllib/core/rl_module/test/test_marl_module.py

```diff
@@ -0,0 +1,115 @@
+import unittest
+
+
+from ray.rllib.core.rl_module.marl_module import MultiAgentRLModule
+from ray.rllib.core.testing.torch.bc_module import DiscreteBCTorchModule
+from ray.rllib.env.multi_agent_env import make_multi_agent
+from ray.rllib.utils.test_utils import check
+
+DEFAULT_POLICY_ID = "default_policy"
+
+
+class TestMARLModule(unittest.TestCase):
```

Comment:
@kouroshHakha Not familiar with the state of other tests, is this test "interesting" or should we remove it from this PR?